src/runtime/pkg/katautils/create.go

```diff
@@ -155,11 +155,18 @@ func CreateSandbox(ctx context.Context, vci vc.VC, ociSpec specs.Spec, runtimeCo
 		}
 	}()
 
-	// Run pre-start OCI hooks.
-	err = EnterNetNS(sandboxConfig.NetworkConfig.NetworkID, func() error {
-		return PreStartHooks(ctx, ociSpec, containerID, bundlePath)
-	})
-	if err != nil {
+	if ociSpec.Annotations == nil {
+		ociSpec.Annotations = make(map[string]string)
+	}
+	ociSpec.Annotations["nerdctl/network-namespace"] = sandboxConfig.NetworkConfig.NetworkID
```

Comment:
@bergwolf, @liubin, @jiangliu - ooi, do you have thoughts on how we're going to manage adding new features like this into the `runtime-rs` version?

Reply:
My first thought would be  "rely on the CI system":)
It would be challenging to keep sync with 2.x branch, so we plan to have runnable 3.0 branch and let the CI system to    disclose missed features.

Reply:
Agreed. The CI is going to be crucially important, but relying on it assumes that _every_ new feature has some sort of test with it, so we're all going to have to be extremely careful to ensure that all PRs do have tests either unit tests or an integration test in the `tests` repo.